functions/src/import-employees-from-alibeez.ts

```diff
@@ -117,7 +117,7 @@ export const importEmployeesFromAlibeez = async (config: Config) => {
   while (leftToSend.length > 0) {
     const nextBatch = leftToSend.splice(
       0,
-      config.features.daily_alibeez_import.batch_size
+      parseInt(config.features.daily_alibeez_import.batch_size, 10)
```

Comment:
There's a function for that in the config module: `asNumber`. You'll need to tweak the input type.

Reply:
I'm not sure about my tweaking, but I think it should be opened to any string since we can't really validate that it is a number until we tried to parse it

Reply:
Sure.